internal/tlsutil/tlsutil_test.go

```diff
@@ -61,12 +61,12 @@ func testfile(t *testing.T, file string) (path string) {
 	return path
 }
 
-func TestNewClientTLS(t *testing.T) {
```

Comment:
Would it be worth restoring the tests for `NewClientTLS` as well? That way we can capture that we are setting renegotiation to `RenegotiateNever` when `NewClientTLS` is called.

Reply:
I would really have liked to actually test the re-negotiation but I don't know of any public place or test (badssl.com) that has re-negotiation enabled. Given that I think the latest change is the best we can do for now.

`

Reply:
Done